src/backend/database/migrations/2025_01_18_100625_create_user_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::create('users', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->integer('company_id')->unsigned();
+            $table->string('email')->unique();
+            $table->string('password');
+            $table->rememberToken();
+            $table->timestamps();
+        });
+    }
+
```

Comment:
If our migrations "just" create the table this will break an existing installation of MPM, right? I.e. the new table will be empty.

Basically, here we would need a step that makes sure the new table is a UNION of all existing tables. Something like `INSERT INTO users ...`

Reply:
> If our migrations "just" create the table this will break an existing installation of MPM, right?

I believe the demo company db will be dropped and reseeded to get things work. Would that be an issue?

Reply:
This totally makes sense. I have updated to accord for this. Please have a look